src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.Types.cs

```diff
@@ -444,6 +452,19 @@ private void AddNameAndTypeArgumentsOrParameters(INamedTypeSymbol symbol)
                 Builder.Add(CreatePart(InternalSymbolDisplayPartKind.Other, symbol, "missing"));
                 AddPunctuation(SyntaxKind.CloseBracketToken);
             }
+
+            return;
+
+            void addExtensionParameter(NamedTypeSymbol? underlyingTypeSymbol)
```

Comment:
>underlyingTypeSymbol [](http://example.com/codeflow?start=56&length=20)

This helper probably should work of off an interface (```ExtensionParameter``` should be part of public API). Prototype comment is sufficient for now.